manifests/custom_hook.pp

```diff
@@ -1,26 +1,44 @@
-# @summary Manage custom hook files within a GitLab project. Custom hooks can be created as a pre-receive, post-receive, or update hook. It's possible to create different custom hook types for the same project - one each for pre-receive, post-receive and update.
+# @summary
+#     Manage custom hook files within a GitLab project.
+#
+# Custom hooks can be created as a pre-receive, post-receive, or update hook.
+# Only one of each is currently supported by this module.
 #
 # @example Custom hook usage
 #   gitlab::custom_hook { 'my_custom_hook':
-#     namespace       => 'my_group',
-#     project         => 'my_project',
-#     type            => 'post-receive',
-#     source          => 'puppet:///modules/my_module/post-receive',
+#     namespace      => 'my_group',
+#     project        => 'my_project',
+#     type           => 'post-receive',
+#     source         => 'puppet:///modules/my_module/post-receive',
+#   }
+#
+# @example Calculate hashed storage path
+#   gitlab::custom_hook { 'my_custom_hook':
+#     project        => 93,
+#     hashed_storage => true,
+#     type           => 'post-receive',
+#     source         => 'puppet:///modules/my_module/post-receive',
 #   }
+#   # Hook path will be `@hashed/6e/40/6e4001871c0cf27c7634ef1dc478408f642410fd3a444e2a88e301f5c4a35a4d`
 #
+# @param project The GitLab project name, or the hashed directory name or project ID number
 # @param namespace The GitLab group namespace for the project.
-# @param project The GitLab project name.
 # @param type The custom hook type. Should be one of pre-receive, post-receive, or update.
-# @param content Specify the custom hook contents either as a string or using the template function. If this paramter is specified source parameter must not be present.
-# @param source Specify a file source path to populate the custom hook contents. If this paramter is specified content parameter must not be present.
+# @param content Specify the custom hook contents either as a string or using the template function. If this paramter is specified source
+#        parameter must not be present.
+# @param source Specify a file source path to populate the custom hook contents. If this paramter is specified content parameter must not
+#        be present.
```

Comment:
please undo those linebreaks

Reply:
Undone, but I am not sure what the issue was, it looks like it rendered fine and the style-checker was complaining about it.

Reply:
the linebreaks in the params are technical fine, they render like they are supposed to. But it just adds a huge unnecessary diff to the commit and doesn't provide any benefit, so we try to avoid it.